README.md

```diff
@@ -146,12 +146,30 @@ The default labels applied to master nodes has changed since Kubernetes 1.11, so
 
 ### Running in an EKS cluster
 
-There is a `job-eks.yaml` file for running the kube-bench node checks on an EKS cluster. **Note that you must update the image reference in `job-eks.yaml`.** Typically you will push the container image for kube-bench to ECR and refer to it there in the YAML file.
+There is a `job-eks.yaml` file for running the kube-bench node checks on an EKS cluster. The significant differences on EKS: Is it's not possible to schedule jobs onto the master node, so master checks can't be performed
 
-There are two significant differences on EKS:
-
-* It uses [config files in JSON format](https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/)
-* It's not possible to schedule jobs onto the master node, so master checks can't be performed
+##### Getting Started with EKS
+1. To create an EKS Cluster refer to [Getting Started with Amazon EKS](https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html) in the *Amazon EKS User Guide*
+  - Information on configuring `eksctl`, `kubectl` and the AWS CLI is within
```

Comment:
I'd rather leave these "getting started with EKS" lines out here because we don't have corresponding "getting started with Kubernetes / OpenShift" etc.

Reply:
Removing that as suggested. Will update PR in a bit @lizrice